packages/mdc-snackbar/foundation.js

```diff
@@ -65,6 +66,14 @@ export default class MDCSnackbarFoundation extends MDCFoundation {
     this.adapter_.deregisterActionClickHandler(this.actionClickHandler_);
   }
 
+  dismissesOnAction() {
```

Comment:
No worries. Looks like the veeeeery last thing is a test for `dismissesOnAction()`.

Then we'll have 100% coverage on these files and all will be right with the universe  :smile: